### ISSUE TITLE
bot not buying super potions correctly

### ISSUE THREAD

**Issue by plxjammerplx** [hectorgimenez/koolo#66]
![mana](https://github.com/hectorgimenez/koolo/assets/41756560/46ab6722-fd31-4dc2-bf30-f6f225888bbc)

![Screenshot 2024-01-27 040150](https://github.com/hectorgimenez/koolo/assets/41756560/c825bab0-7158-4c6d-b5a8-2f68c1b4d5af)
I keep running into the issue where the bot would always point to the last bottom slot in the last column of npcs in attempt to buy super potions but buy mana pots correctly.  Even though it's says it's buying super healing potions, it's just clicking on the bottom last slot where it's empty for both Act 1 and Act 2.

Should be: Purchased SuperHealingPotion [X:9 Y:6]

**Comment by Elanoran** [comment]
I have tried that as well, make sure the window size is right, can also try to alt+shift, then alt+shift again, to resize window and sometimes a reboot works

**Comment by plxjammerplx** [comment]
> I have tried that as well, make sure the window size is right, can also try to alt+shift, then alt+shift again, to resize window and sometimes a reboot works

I have tried that it's clicking everything else perfectly just besides buying healing potions in Act 1 and Act 2. I have tried changing my desktop resolution to 16:9 1440p, 1080p and 720p from my standard 21:9 1440p resolution already as well. I have tested both game resolution 1080p and 720p windowed mode too. Act 1 and Act 2 have wrong healing potion pointers for free.
